src/transformers/generation/utils.py

```diff
@@ -1297,6 +1297,28 @@ def _validate_generated_length(self, generation_config, input_ids_length, has_de
                     UserWarning,
                 )
 
+    def _extend_attention_mask(self, model_kwargs: Dict[str, Any], to: int) -> Dict[str, Any]:
+        if self.config.is_encoder_decoder:
+            key = "decoder_attention_mask"
+        else:
+            key = "attention_mask"
+
+        if key not in model_kwargs:
+            return model_kwargs
+
+        mask = model_kwargs[key]
+        amount = to - mask.shape[1]
```

Comment:
```suggestion
        mask_extension_length = new_mask_length - mask.shape[1]
```
nit: `amount` is a bit vague. We prefer long, well-defined variable names, as they facilitate maintenance 
(This change needs to be propagated below)